tests/docs-screenshots.spec.ts

```diff
@@ -91,20 +111,617 @@ test("instances", async ({ page }) => {
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/forward_create_ovn.png",
+    clip: getClipPosition(240, 0, 1440, 760),
+  });
+
+  await page.getByRole("link", { name: "Networks", exact: true }).click();
+  await page.getByRole("button", { name: "Create network" }).click();
+  await page.getByRole("heading", { name: "Create a network" }).click();
+  await page.getByLabel("Type").selectOption("bridge");
+  await page.getByPlaceholder("Enter name").fill(network1);
+  await page.getByRole("button", { name: "Create", exact: true }).click();
+  await page.getByTestId("notification-close-button").click();
+  await page.getByRole("link", { name: network1 }).first().click();
+  await page.waitForSelector(`text=IPv4 address`);
+  await page.getByText("/24").getByRole("button").click();
+  networkSubnet = await page.inputValue("input#ipv4_address");
+  listenAddress = networkSubnet.replace("1/24", "1");
+  targetAddress = networkSubnet.replace("1/24", "3");
+  await page.getByTestId("tab-link-Forwards").click();
+  await page.getByRole("link", { name: "Create forward" }).click();
+  await page.getByLabel("Listen address").fill(listenAddress);
+  await page.getByLabel("Default target address").fill(targetAddress);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/forward_create.png",
+    clip: getClipPosition(240, 0, 1440, 760),
+  });
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/forward_create_bridge.png",
+    clip: getClipPosition(240, 0, 1440, 760),
+  });
+
+  await page.getByRole("button", { name: "Add port", exact: true }).click();
+  await page.getByLabel("Port 0 listen port").fill("80");
+  await page.getByLabel("Port 0 target port").fill("443");
+  await page.getByLabel("Port 0 target address").fill(targetAddress);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/forward_create_port.png",
+    clip: getClipPosition(240, 0, 1440, 760),
+  });
+
+  await page.getByLabel("Delete port").click();
+  await page.getByRole("button", { name: "Create" }).click();
+  await page.getByTestId("notification-close-button").click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/forward_delete.png",
+    clip: getClipPosition(240, 0, 1440, 360),
+  });
+
+  await page.getByRole("link", { name: "Edit network forward" }).click();
+  await page.getByRole("button", { name: "Add port" }).click();
+  await page.getByLabel("Port 0 listen port").fill("80");
+  await page.getByLabel("Port 0 target port").fill("443");
+  await page.getByLabel("Port 0 target address").fill(targetAddress);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/forward_edit_ex2.png",
+    clip: getClipPosition(240, 0, 1440, 760),
+  });
+
+  await page.getByRole("button", { name: "Update" }).click();
+  await page.getByTestId("notification-close-button").click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/forward_edit_ex1.png",
```

Comment:
This screen is very different. I think we need to wait after the update to ensure we are on the right page:

<img width="1922" height="959" alt="image" src="https://github.com/user-attachments/assets/44a2f14c-391c-417b-8856-a6ef4499c698" />